nni/common/graph_utils.py

```diff
@@ -709,7 +709,7 @@ def _build_graph(self):
 
         self.leaf_modules = self._extract_leaf_modules()
         module_to_type = {name: parse_traced_name(
-            module._name) for name, module in self.trace.named_modules()}
+            module.original_name) for name, module in self.trace.named_modules()}
```

Comment:
what is the difference between `_name` and `original_name`?

Reply:
`RecursiveScriptModule` don't have `_name`, and I think they have no difference in our scenario. @zheng-ningxin please help to check.

```
        @property
        def original_name(self):
            if type(self) == str(self._c._type().name()):
                return ""
            return str(self._c._type().name())
```

```
self.__dict__["_name"] = type(orig).__name__
```

Reply:
It can actually be an if condition, where if the module type is `RecusiveScriptModule` we ask for `original_name`, else use `name`

Reply:
Also suggest to an `if` condition here.

Reply:
@shenoynikhil98 @zheng-ningxin
good suggestion, have fix it.